### PR TITLE
feat(json-schema-form): add enum keyword with string type

### LINKED ISSUE #66: [Json Schema Form] Improve Component(generate_id, enum, password, json(object), order, markdown, disabled)
### 1. enum Keyword
- SearchDropdown(single select) 

### ~~2. Array Type~~
- ~~Array: TextInput(multi input)~~
- ~~Array + Enum: SearchDropdown~~

### 3. Extend Formats
- generate_id
- password
- json

### 4. Custom Keyword
- markdown

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [ ] Tested with console

#### Add enum keyword for string type only

https://user-images.githubusercontent.com/26986739/192254780-cf723148-162f-4be5-a8c0-dee635218d4a.mov

### CODE REVIEW

File: src/inputs/forms/new-json-schema-form/helper.ts [hunk 1]
```diff
@@ -56,3 +63,21 @@ export const getInputTypeBySchemaProperty = (schemaProperty: InnerJsonSchema): T
...
 
 export const getInputPlaceholderBySchemaProperty = (schemaProperty: InnerJsonSchema) => schemaProperty.examples?.[0] ?? '';
+
+export const getMenuItemsBySchemaProperty = (schemaProperty: InnerJsonSchema): SelectDropdownMenu[]|undefined => {
+    if (Array.isArray(schemaProperty.enum)) {
+        try {
+            return schemaProperty.enum.map((d) => {
+                if (typeof d === 'string') {
+                    return { name: d, label: d };
+                }
```
Comment (yuda110) [inline src/inputs/forms/new-json-schema-form/helper.ts:73]: Are enum items always strings?

**Review by yuda110 — APPROVED** [review]
LGTM! I left a question.

File: src/inputs/forms/new-json-schema-form/helper.ts (hunk 1 above)
Comment (WANZARGEN) [reply src/inputs/forms/new-json-schema-form/helper.ts:73]: yes. Jong said only string!

**Pull request merged**